### DISCUSSION TITLE
Does the public database instance still work?

### DISCUSSION THREAD

**Discussion by bytrangle** [formbricks/formbricks#67] [General Requests] [answered]
Hello Snoopforms team. You shared a public database isntance [here](https://github.com/snoopForms/snoopforms#getting-started).

I got an error as soon as I ran the `docker run` command:

```
docker: Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:5432 -> 0.0.0.0:0: listen tcp 0.0.0.0:5432: bind: address already in use.
```

I will read up how to set up databases, but asking just in case 😅.

**Comment by bytrangle** [comment, marked as answer]
After much searching, I realized that there were a few processes running at port 5432 that I didn't know. It probably has to do with MacOS Monterey since I wasn't the only experiencing this issue. [Here's](https://stackoverflow.com/a/72010271/12160227) the answer that helped me get a database up and running with Docker.

Thank you for the quick and dirty database.

**Reply by mattinannt** [reply]
@bytrangle great that you solved the problem. Yesterday I also tried to debug this and finding a solution for you but wasn't really able to provide you with good advice. Great that you found it yourself. If you have any other questions, always feel free to ask; the fastest way for that is often our Discord server: [https://snoopforms.com/discord](https://snoopforms.com/discord)
